components/datetime/benches/fixtures/tests/styles.json

```diff
@@ -87,7 +69,7 @@
                 "options": {
                     "style": {
                         "date": "long",
-                        "time": "long"
+                        "time": "short"
```

Comment:
`DateTimeFormat` no longer supports patterns with time zones, so it cannot use `Time::Long` or `Time::Full`.

These are benchmarked by `ZonedDateTimeFormat` which pulls from `styles_with_zones.json`.